src/main/java/com/softserve/actent/model/entity/Event.java

```diff
@@ -0,0 +1,124 @@
+package com.softserve.actent.model.entity;
+
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import com.softserve.actent.constant.StringConstants;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.validator.constraints.Length;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.ForeignKey;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.Set;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = "events")
+public class Event {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NonNull
+    @NotBlank(message = StringConstants.TITLE_SHOULD_NOT_BE_BLANK)
+    @Length(max = 100, message = "Too long")
+    @Column(nullable = false, length = 100)
+    private String title;
+
+    @Length(max = 500, message = "Too long")
+    @Column(length = 500)
+    private String description;
+
+    @CreationTimestamp
+    @Column(name = "creation_date")
+    private Timestamp creationDate;
```

Comment:
Change type to java.time.LocalDateTime. Also this field can not be nullable.

Reply:
Why?

Reply:
Why what? It's way more comfortable to work with java.time than with sql timestamp, and every event should have creation date, so it definitely is NOT NULL.

Reply:
This annotation work with timestamp only

Reply:
https://docs.jboss.org/hibernate/orm/4.3/javadocs/org/hibernate/annotations/CreationTimestamp.html

You are not right, proof in documentation by link.

Reply:
Ok) Each better for us?

Reply:
java.time.LocalDateTime, I guess. Here is up-to-date documentation page: https://docs.jboss.org/hibernate/orm/5.2/javadocs/org/hibernate/annotations/CreationTimestamp.html